### PR TITLE
[IOTDB-3844] Clear up Non-Seed-ConfigNode register process

### PR DESCRIPTION
**Pull request by CRZbulabula**
Updates:

1. It's enough to use TSStatus as registerConfigNode interface's response
2. Rename the input parameter of addConsensusGroup interface to TAddConsensusGroupReq

And these modifications are safe:

image

### CODE REVIEW

**Review by RYH61 — APPROVED** [review]
LGTM~

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java [hunk 1]
```diff
@@ -698,103 +697,65 @@ public TPermissionInfoResp checkUserPrivileges(
...
     if (status.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-      TConfigNodeRegisterResp errorResp1 = checkConfigNodeRegisterResp(req);
-      if (errorResp1 != null) return errorResp1;
+      TSStatus errorStatus = checkConfigNodeGlobalConfig(req);
+      if (errorStatus != null) {
+        return errorStatus;
+      }
 
       procedureManager.addConfigNode(req);
-      return nodeManager.registerConfigNode(req);
```
Comment (wangchao316) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:710]: nodemanager does not registerConfigNode?

Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:710]: The nodeManager.registerConfigNode was used to get the current PartitionRegionId and ConfigNodeList. However, the purpose of registerConfigNode is only to activate the addConfigNodeProcedure, thus we don't need to get these information anymore. Therefore, I just delete this redundant interface.

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**